internal/stubserver/stubserver.go

```diff
@@ -56,9 +56,11 @@ type StubServer struct {
 	testgrpc.TestServiceServer
 
 	// Customizable implementations of server handlers.
-	EmptyCallF      func(ctx context.Context, in *testpb.Empty) (*testpb.Empty, error)
-	UnaryCallF      func(ctx context.Context, in *testpb.SimpleRequest) (*testpb.SimpleResponse, error)
-	FullDuplexCallF func(stream testgrpc.TestService_FullDuplexCallServer) error
+	EmptyCallF                func(ctx context.Context, in *testpb.Empty) (*testpb.Empty, error)
+	UnaryCallF                func(ctx context.Context, in *testpb.SimpleRequest) (*testpb.SimpleResponse, error)
+	FullDuplexCallF           func(stream testgrpc.TestService_FullDuplexCallServer) error
+	ClientStreamingInputCall  func(stream testgrpc.TestService_StreamingInputCallServer) error
+	ClientStreamingOutputCall func(req *testpb.StreamingOutputCallRequest, stream testgrpc.TestService_StreamingOutputCallServer) error
```

Comment:
@janardhanvissa let's change this StreamingInputCall and StreamingOutputCall and in the comment mention that that input is for ClientStreaming and output is for ServerStreaming